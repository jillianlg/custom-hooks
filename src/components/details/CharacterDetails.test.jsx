import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterDetail from './CharacterDetails';

describe('CharacterDetails component', () => {
  afterEach(() => cleanup());
  it('renders character data by id', () => {
    const { asFragment } = render(
      <CharacterDetail
        name="Morty Smith"
        image="morty-smith.url"
        status="Alive"
        species="Human"
      />);
    expect(asFragment()).toMatchSnapshot();
  });
});
