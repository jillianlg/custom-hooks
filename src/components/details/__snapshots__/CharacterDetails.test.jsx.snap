// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetails component renders character data by id 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt="Morty Smith"
      src="morty-smith.url"
    />
    <figcaption>
      <p>
        Morty Smith
      </p>
      <p>
        Alive
      </p>
      <p>
        Human
      </p>
    </figcaption>
  </figure>
</DocumentFragment>
`;
